src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCountTableColumns.java

```diff
@@ -17,6 +18,31 @@ public String toString() {
         return columnName;
     }
 
-    public static final String[] COLUMN_NAME_ARRAY =
-            Stream.of(values()).map(AllelicCountTableColumns::toString).toArray(String[]::new);
+    public static final String[] BASIC_COLUMN_NAME_ARRAY = {
```

Comment:
Please change  ```String[]``` to ```List<String>```. The reason is that while the array constant reference is final, the content of the array could be changed. Please use ```Collections.unmodifiableList()``` to make it impossible to alter it in runtime.